index.html

```diff
@@ -20,18 +26,20 @@
               <div class="navbar_menu_button">
                 <img src="assets/menu_icon.svg" alt="menu_icon">
               </div>
-              <ul class="navbar_menu_content">
-                  <li class="navbar_menu_content_element">Portofolio</li>
-                  <li class="navbar_menu_content_element">About me</li>
-                  <li class="navbar_menu_content_element">Contact</li>
-              </ul>
             </div>
+            <ul class="navbar_menu_content visible_nav_menu hidden">
+                  <li class="close_menu_button"> <i class="fa-solid fa-xmark close_button"></i></li>
+                  <a href="#hello"><li class="navbar_menu_content_element">Hello</li></a>
```

Comment:
- [ ] I would like to recommend you replace the option name **Hello** with **home** because it leads to the home page :+1: